### PR TITLE
UI: Improve performance of chat group query

### PR DESCRIPTION
**Pull request by LucianBuzzo**
Only return the fields required in the chat group query

Change-type: patch
Signed-off-by: Lucian Buzzo <lucian.buzzo@gmail.com>

***

**Please remember to write tests for your changes**. We aim to automatically
deploy `master` to production, and we can't safely do this without a solid
test suite!

### CODE REVIEW

File: apps/ui/lib/store/actioncreators/index.ts [hunk 1]
```diff
@@ -543,6 +548,9 @@ export const actionCreators = {
 				]).then(async ([loops, orgs, types, relationships, groups, config]) => {
 					const state = getState();
 
+					console.log({ allGroupsWithUsersQuery });
+					console.log({ groups });
+
```
Comment (aethernet) [inline apps/ui/lib/store/actioncreators/index.ts:553]: you forgot those

**Review by aethernet — COMMENTED** [review]
you left a console.log
this reduces the payload to 167k (instead of 67m), well done.

File: apps/ui/lib/store/actioncreators/index.ts (hunk 1 above)
Comment (LucianBuzzo) [reply apps/ui/lib/store/actioncreators/index.ts:553]: resolved

**Comment by LucianBuzzo** [comment]
I've removed the logs now, I think we hit some intermittent test failure. Rebuilding now

**Comment by LucianBuzzo** [comment]
@balena-ci I self-certify!

**Pull request merged**